ibm-cloud-pak-for-data/scripts/deployCloudPakDataWkc.sh

```diff
@@ -0,0 +1,73 @@
+export OCUSER=$1
+export OCPASSWD=$2
+export MASTERPUBLICHOSTNAME=$3
+export NAMESPACE=$4
+export APIKEYUSERNAME=$5
+export APIKEY=$6
+export STORAGEOPTION=$7
+export INSTALLERARTIFACTSLOCATION="https://prodcpdartifacts.blob.core.windows.net"
```

Comment:
remember to pull all artifacts required for deployment into the sample - if they are large, we'll host them in the CI subscription - won't comment on the rest